src/setupTests.js

```diff
@@ -5,12 +5,11 @@ configure({ adapter: new Adapter() });
 
 class LocalStorageMock {
   constructor() {
-    this.store = {
-    };
+    this.store = {};
   }
 
   getItem(key) {
-    return this.store[key] || false;
+    return this.store[key] || null;
```

Comment:
@lytes20, I think that it still does the same thing.

Reply:
@lytes20 , It does the same thing and I have changed @abulojoshua1 's file that needs it to conform. Mine is also more semantically correct, since you can either have a value or a null.